src/temp/Explore/Explore.module.scss

```diff
@@ -0,0 +1,98 @@
+@import '../../theme/theme';
+.exploreMainContainer {
+  max-width: 182px;
+  background-color: $backgroundColor;
+  border-radius: 12px;
+  margin-top: 10px;
+  margin-left: 3px;
+  padding-top: 14px;
+  padding-bottom: 14px;
+}
+
+.exploreContainer {
+  width: 150px;
+  margin-left: 16px;
+}
+
+.ecosystemMainContainer {
+  height: 14px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.ecosystemContainer {
+  display: flex;
+  font-size: $fontXXSmall;
+  font-weight: 400;
+  align-items: center;
+  gap: 4px;
+  text-transform: capitalize;
+}
+
+.switchContainer {
+  height: 14px;
+  width: 24px;
+}
+
+.infoIconConatiner {
+  margin-top: 2px;
+}
+
+.rangeMainContainer {
+  width: 150px;
+  height: 30px;
+  background-color: rgba(var(--deforestration-range-background-new), 0.1);
+  margin-top: 6px;
+  border-radius: 6px;
+  position: relative;
+}
+.rangeContainer {
+  width: 131px;
+  padding-top: 8px;
+  position: absolute;
+  left: 8px;
+  top: 6px;
+}
+
+.playIconContainer {
+  position: absolute;
+  bottom: -5px;
+}
+
+.sliderContainer {
+  width: 107px;
+  position: absolute;
+  left: 20px;
+  bottom: -7px;
+}
```

Comment:
The styles here frequently use `absolute` positioning, as a result individual adjustments are likely to be needed if the Explore container width changes.